### ISSUE TITLE
Option to remove play button

### ISSUE THREAD

**Issue by viktor-evdokimov** [NascHQ/nasc-vscode-mac-touchbar#9]
Can we add option to remove play button?

**Comment by felipenmoura** [comment]
I surely hope so!
Actually, those buttons are added there and there is even an issue on VSCode repo talking about making them optional.
https://github.com/Microsoft/vscode/issues/38333

I believe it's already there, we could think of a way we could do it ourselves (from within this plugins)

**Comment by pauloschilling** [comment]
Microsoft/vscode#38333 was closed (completed) and the only thing they did was adding a new config (`keyboard.touchbar.enabled`) to **completely disable** the Touch Bar support.

_É bucha..._ 🤦‍♂️

**Comment by debitan** [comment]
I've also been looking for a way to do this. I don't find the default touchbar buttons on VSCode useful.

**Comment by felipenmoura** [comment]
yup, still waiting for a way to do that! It would be great, we would have much more space to deal with, right?!

**Comment by bpasero** [comment]
Landed in VSCode via https://github.com/microsoft/vscode/pull/70174
